phpBB/styles/prosilver/template/overall_footer.html

```diff
@@ -11,6 +11,7 @@
 		{CREDIT_LINE}
 		<!-- IF TRANSLATION_INFO --><br />{TRANSLATION_INFO}<!-- ENDIF -->
 		<!-- EVENT overall_footer_copyright_append -->
+		<br><a href="{U_PRIVACY}" title="{L_PRIVACY}" role="menuitem"><span>{L_PRIVACY}</span></a>&nbsp;&vert;&nbsp;<a href="{U_TERMS_USE}" title="{L_TERMS_USE}" role="menuitem"><span>{L_TERMS_USE}</span></a>
```

Comment:
just use a paragraph tag we should not use br its not style/themeable.

did we already have these lang strings? I would just say "privacy" and "terms" this is the common representation of these, also why not use twig?

Reply:
These already existed. We could also add these shorter versions as I was thinking the same thing.

Reply:
Fwiw, I wouldn't change this to a <p> right now. Using <p> adds a whole new set of issues like weird line-height, margin-bottom, etc. As this fits with what we're currently using I'd keep it like that and just change the links to have shorter titels.